Source/Blazorise/Components/Tabs/Tabs.razor.cs

```diff
@@ -221,6 +221,19 @@ public TabPosition TabPosition
             }
         }
 
+        /// <summary>
+        /// Gets or sets the TabsMode.
+        /// </summary>
+        [Parameter]
+        public TabsMode Mode
+        {
+            get => state.Mode;
+            set
+            {
+                state = state with { Mode = value };
+            }
```

Comment:
missing `DirtyClasses();`

Reply:
Does it need to Dirty classes here??

Reply:
You never know. Maybe some provider might need it so it is good to have it. Especially when you update the `state` object.

Reply:
I see, OK.